pyproject.toml

```diff
@@ -8,3 +8,50 @@ known_first_party = ["myst_parser", "tests"]
 known_third_party = ["docutils", "markdown_it", "sphinx"]
 # Group first party and local folder imports together
 no_lines_before = "LOCALFOLDER"
+
+[tool.towncrier]
+  directory = "docs/changelog-fragments.d/"
+  filename = "CHANGELOG.md"
+  issue_format = "{{issue}}`{issue}`"
```

Comment:
This is what's rendered in the template, the syntax is basically what works in f-strings. It produces a role targeting an issue that would later be rendered by Sphinx but some projects use an explicit external URL syntax instead.